lib/instrumental/agent.rb

```diff
@@ -473,6 +479,7 @@ def run_worker_loop
         # or we cannot reach the server
         # or the connection state of this socket is in a race
         logger.error "unable to connect to Instrumental, hanging up with #{@queue.size} messages remaining"
+        logger.debug "Exception: #{err.inspect}\n#{err.backtrace.join("\n")}"
```

Comment:
Yeah, it was helpful when working on the tests and I think if your log level is debug it's reasonable to see what exception actually occurred, which otherwise wouldn't be shown.